spec/std/spec_spec.cr

```diff
@@ -41,6 +41,30 @@ describe "Spec matchers" do
     end
   end
 
+  describe "should_have" do
+    it "passes when string includes? specified substring" do
+      "hello world!".should have String, includes?("world")
```

Comment:
@jhass I can't get rid of this `String` here. Any ideas?

This is needed because at the time `have` expanded, it doesn't know anything about `T` type parameter of `HaveExpectation` class. And compiler spits out error, that `T` is undefined constant.